ReactAndroid/src/main/java/com/facebook/react/uimanager/TouchTargetHelper.java

```diff
@@ -115,31 +114,38 @@ private static View findTouchTargetView(float[] eventCoords, ViewGroup viewGroup
     int childrenCount = viewGroup.getChildCount();
     // Consider z-index when determining the touch target.
     ReactZIndexedViewGroup zIndexedViewGroup =
-        viewGroup instanceof ReactZIndexedViewGroup ? (ReactZIndexedViewGroup) viewGroup : null;
+            viewGroup instanceof ReactZIndexedViewGroup ? (ReactZIndexedViewGroup) viewGroup : null;
+    String overflow = viewGroup instanceof ReactClippingViewGroup
+            ? ((ReactClippingViewGroup) viewGroup).getOverflow()
+            : null;
+    boolean allowOverflow = overflow == null || overflow.equals(ViewProps.VISIBLE);
+
     for (int i = childrenCount - 1; i >= 0; i--) {
       int childIndex =
           zIndexedViewGroup != null ? zIndexedViewGroup.getZIndexMappedChildIndex(i) : i;
       View child = viewGroup.getChildAt(childIndex);
-      PointF childPoint = mTempPoint;
+      TouchTestResult touchTestData = mTouchTestData;
+
       if (isTransformedTouchPointInView(
-          eventCoords[0], eventCoords[1], viewGroup, child, childPoint)) {
+          eventCoords[0], eventCoords[1], viewGroup, child, touchTestData, allowOverflow)) {
         // If it is contained within the child View, the childPoint value will contain the view
         // coordinates relative to the child
         // We need to store the existing X,Y for the viewGroup away as it is possible this child
         // will not actually be the target and so we restore them if not
         float restoreX = eventCoords[0];
         float restoreY = eventCoords[1];
-        eventCoords[0] = childPoint.x;
-        eventCoords[1] = childPoint.y;
-        View targetView = findTouchTargetViewWithPointerEvents(eventCoords, child);
+        eventCoords[0] = touchTestData.x;
+        eventCoords[1] = touchTestData.y;
+        View targetView = findTouchTargetViewWithPointerEvents(eventCoords, child, touchTestData.hittest);
```

Comment:
```suggestion
        View targetView = findTouchTargetViewWithPointerEvents(eventCoords, child, mTouchTestData.hittest);
```

Reply:
Why do you care if it has a local variable to point to the actual static one?
Did you generate release mode code and investigate it to see if it actually costs something?

Reply:
@danielgindi never mind

Reply:
@danielgindi I am learning this now as I'm a Ruby Developer. 

>Why do you care if it has a local variable to point to the actual static one?

As you can read [here](https://stackoverflow.com/a/70358/11452286) static nested class, means the class is saved as a static property of the parent, in your case would be `TouchTargetHelper.TouchTestResult`

>Static nested classes are accessed using the enclosing class name:

```java
TouchTargetHelper.TouchTestResult
```
>For example, to create an object for the static nested class, use this syntax: (from outside that class)

```java
TouchTargetHelper.TouchTestResult nestedObject = new TouchTargetHelper.TouchTestResult();
```
My comment has nothing to do with the above and I believe you misunderstood my point.

I am not referring to the `static class TouchTestResult`, but how you are using the local and instance variables of the TouchTargetHelper class. 

>Did you generate release mode code and investigate it to see if it actually costs something?

With my regards
I wish you a good day
Fabrizio Bertoglio